app/controllers/api/v2/courses_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
         authorize provider
 
         render jsonapi: provider.courses,
-               class: SERIALIZABLE_CLASSES
+               class: { Course: API::V2::CourseSerializable }
```

Comment:
I meant something just like this

```ruby
render jsonapi: provider.courses
```

omitting the need for `class`

see the hook
https://github.com/jsonapi-rb/jsonapi-rails/blob/14a9421658495fe916ec1886834b6adfd764b211/lib/jsonapi/rails/controller/hooks.rb#L13

The thing I would of expected is either conforming to a naming convention.

If you look at this test its don's specify a 'class' and the json returns contain the fields

Reply:
> omitting the need for `class`

Good find @defong. 💯 I've made a change that should sort this out.

Reply:
nice one, I had no idea how to approach it, :upside_down_face: